tests/seasons.test.ts

```diff
@@ -230,6 +220,34 @@ describe('Testing date range functions', () => {
         }
       });
     });
+
+    describe('Check weekdays before and after Epiphany', () => {
+      const romcal = new Romcal({ epiphanyOnSunday: true });
+
+      test('In 2021 Epiphany is January 3, so there is 1 weekday before, and 6 weekdays after', () => {
+        const datesBefore2021 = romcal.dates(2021).allDatesBeforeEpiphany();
+        expect(datesBefore2021.length).toEqual(1);
+
+        const datesAfter2021 = romcal.dates(2021).allDatesAfterEpiphany();
+        expect(datesAfter2021.length).toEqual(6);
+      });
+
+      test('In 2022 Epiphany is January 2, so there is only 6 weekdays after', () => {
+        const datesBefore2022 = romcal.dates(2022).allDatesBeforeEpiphany();
+        expect(datesBefore2022.length).toEqual(0);
+
+        const datesAfter2022 = romcal.dates(2022).allDatesAfterEpiphany();
+        expect(datesAfter2022.length).toEqual(6);
+      });
+
+      test('In 2023 Epiphany is January 3, and the Baptism is the day after, so there is only 6 weekdays before', () => {
```

Comment:
```suggestion
      test('In 2023, Epiphany occurs on January 3, and the Baptism of the Lord is on the following day, so there are only 6 weekdays before', () => {
```

Again, it might be a good idea to explicitly mention that _there are no weekdays after_. :wink: